src/complex.rb

```diff
@@ -0,0 +1,78 @@
+class Complex
+
+  def terrible_idea(param)
+    Model.find_by_sql("SELECT #{param}")
+  end
```

Comment:
Bliss.AI: report

| Debt | Issue |
|---:|---|
| 24.5 | Cyclomatic complexity for get_cost_per_line is too high. [51/6] |
| 24.5 | Perceived complexity for get_cost_per_line is too high. [52/7] |
| 7.4 | Assignment Branch Condition size for get_cost_per_line is too high. [89.31/15] |
| 4.5 | Method has too many lines. [55/10] |